doc/dev/debugging.rst

```diff
@@ -403,4 +403,42 @@ The recompilation starts after the given checkpoint stage.
 
     replace_ir(f, "HLOLoweringPass", new_ir)
     f(2.0)
-    >> 8.0
\ No newline at end of file
+
+C Executable Generation
+=================
+
+Catalyst provides a way to generate a c executable that calls the shared object of the given
+qjit-decorated function.
+In ``catalyst.debug.compiler_functions``, ``compile_executable(fn: QJIT, *args) -> str, str`` takes
+a qjit-decorated function and arguments for that function.
+It returns a string of LD_LIBRARY_PATH and the path to the output executable file.
```

Comment:
Did we not settle on a solution that fixes up the rpath rather than relying on environment variables? (note `LD_LIBRARY_PATH` is linux only)

Reply:
Yes. I missed this part. Just fixed it.